ivy/functional/frontends/torch/spectral_ops.py

```diff
@@ -0,0 +1,19 @@
+import ivy
+from ivy.functional.frontends.torch.func_wrapper import (
+    to_ivy_arrays_and_back,
+)
+
+
+@to_ivy_arrays_and_back
+def hann_window(
+    window_length,
+    periodic=True,
+    *,
+    dtype=None,
+    device,
+    layout=ivy.strides,
+    required_grad=False
```

Comment:
The kwarg `required_grad` should be `requires_grad`

Reply:
I think we don't need it.. As the backend of ivy doesn't  require `requires_grad`